doc/reference/cmdline.rst

```diff
@@ -395,31 +388,66 @@ customize the amount of logging output
         In case of conflict between :option:`--log-level` and
         :option:`--log-handler`, the latter is used
 
-emails
-------
+.. _reference/cdmline/workers:
 
-.. option:: --email-from <address>
+Multiprocessing
+'''''''''''''''
 
-    Email address used as <FROM> when Odoo needs to send mails
+.. option:: --workers <count>
 
-.. option:: --smtp <server>
+    if ``count`` is not 0 (the default), enables multiprocessing and sets up
+    the specified number of HTTP workers (sub-processes processing HTTP
+    and RPC requests).
 
-    Address of the SMTP server to connect to in order to send mails
+    .. note:: multiprocessing mode is only available on Unix-based systems
 
-.. option:: --smtp-port <port>
+    A number of options allow limiting and recycling workers:
 
-.. option:: --smtp-ssl
+    .. option:: --limit-request <limit>
 
-    If set, odoo should use SSL/STARTSSL SMTP connections
+        Number of requests a worker will process before being recycled and
+        restarted.
 
-.. option:: --smtp-user <name>
+        Defaults to *8196*.
 
-    Username to connect to the SMTP server
+    .. option:: --limit-memory-soft <limit>
 
-.. option:: --smtp-password <password>
+        Maximum allowed virtual memory per worker. If the limit is exceeded,
+        the worker is killed and recycled at the end of the current request.
 
-    Password to connect to the SMTP server
+        Defaults to *2048MiB*.
+
+    .. option:: --limit-memory-hard <limit>
+
+        Hard limit on virtual memory, any worker exceeding the limit will be
+        immediately killed without waiting for the end of the current request
+        processing.
+
+        Defaults to *2560MiB*.
+
+    .. option:: --limit-time-cpu <limit>
+
+        Prevents the worker from using more than <limit> CPU seconds for each
+        request. If the limit is exceeded, the worker is killed.
+
+        Defaults to *60*.
+
+    .. option:: --limit-time-real <limit>
+
+        Prevents the worker from taking longer than <limit> seconds to process
+        a request. If the limit is exceeded, the worker is killed.
+
+        Differs from :option:`--limit-time-cpu` in that this is a "wall time"
```

Comment:
@Feyensv this sentence feels a bit odd to me.

Reply:
Well, it is the same sentence but moved in the file :D.
As it says just after, it seems to mean that this option limits the total processing time of a request, not only the cpu time (which can be specified by --limit-time-cpu).